libethereum/BlockQueue.cpp

```diff
@@ -389,7 +389,7 @@ void BlockQueue::tick()
 
 		cblockq << "Checking past-future blocks...";
 
-		unsigned t = time(0);
+		unsigned t = utcTime();
```

Comment:
Do not trunc the value. The end of 32-bit Unix time is coming: https://en.wikipedia.org/wiki/Year_2038_problem.